lib/libc/gen/tls_malloc.c

```diff
@@ -101,6 +109,18 @@ union	overhead {
 #define	ov_index	ovu.ovu_index
 };
 
+struct pagepool_header {
+	size_t			 ph_size;
+#ifdef __CHERI_PURE_CAPABILITY__
+	size_t			 ph_pad1;
```

Comment:
Why?

Reply:
This is just explicit padding. It's not strictly needed.